pal_tiago/tiago.xml

```diff
@@ -0,0 +1,222 @@
+          <geom type="mesh" density="0" rgba="1 1 1 1" mesh="arm_1"/>
+          <body name="arm_2_link" pos="0.125 0.0195 -0.031" quat="0.707107 0.707107 0 0">
+            <inertial pos="0.030432 0.000229 -0.005942" quat="0.126897 0.72859 0.136233 0.659161" mass="1.8004"
+              diaginertia="0.00437229 0.00432701 0.0017837"/>
+            <joint name="arm_2_joint" pos="0 0 0" axis="0 0 1" range="-1.5708 1.09083" damping="10" frictionloss="1"/>
+            <geom type="mesh" density="0" rgba="0.1 0.1 0.1 1" mesh="arm_2"/>
+            <body name="arm_3_link" pos="0.0895 0 -0.0015" quat="0.5 -0.5 -0.5 0.5">
+              <inertial pos="0.007418 -0.004361 -0.134194" quat="0.516615 0.0556881 0.00536911 0.854388" mass="1.8"
+                diaginertia="0.0200771 0.0196154 0.00264853"/>
+              <joint name="arm_3_joint" pos="0 0 0" axis="0 0 1" range="-3.53429 1.5708" damping="10" frictionloss="1"/>
+              <geom type="mesh" density="0" rgba="0.1 0.1 0.1 1" mesh="arm_3"/>
+              <body name="arm_4_link" pos="-0.02 -0.027 -0.222" quat="0.5 -0.5 -0.5 -0.5">
+                <inertial pos="-0.095658 0.014666 0.018133" quat="-0.161572 0.615556 -0.202229 0.744371" mass="1.4327"
+                  diaginertia="0.00906004 0.00831328 0.00177669"/>
+                <joint name="arm_4_joint" pos="0 0 0" axis="0 0 1" range="-0.392699 2.35619" damping="10"
+                  frictionloss="1"/>
+                <geom type="mesh" density="0" rgba="0.1 0.1 0.1 1" mesh="arm_4"/>
+                <body name="arm_5_link" pos="-0.162 0.02 0.027" quat="0.707107 0 -0.707107 0">
+                  <inertial pos="0.001078 -0.000151 -0.077173" quat="0.999999 -0.000362286 -0.00167006 0.000217394"
+                    mass="0.935914" diaginertia="0.00103619 0.000794527 0.000439824"/>
+                  <joint name="arm_5_joint" pos="0 0 0" axis="0 0 1" range="-2.0944 2.0944" damping="1" frictionloss="1"/>
+                  <geom type="mesh" density="0" rgba="0.9 0.9 0.9 1" mesh="arm_5-wrist-2017"/>
+                  <body name="arm_6_link" pos="0 0 0.15" quat="0.5 -0.5 -0.5 -0.5">
+                    <inertial pos="-0.000153 -0.003122 0.000183" quat="0.510936 0.49453 0.508164 0.485956"
+                      mass="0.302758" diaginertia="3.85419e-05 3.33205e-05 3.29226e-05"/>
+                    <joint name="arm_6_joint" pos="0 0 0" axis="0 0 1" range="-1.41372 1.41372" damping="1"
+                      frictionloss="1"/>
+                    <geom type="mesh" density="0" rgba="0.9 0.9 0.9 1" mesh="arm_6-wrist-2017"/>
+                    <body name="arm_7_link" quat="0.5 0.5 0.5 0.5">
+                      <inertial pos="-0.000173894 0.000176395 0.0817355"
+                        quat="0.997803 -0.00177536 -0.00128747 0.0662101" mass="1.00276"
+                        diaginertia="0.0025841 0.0022568 0.000746434"/>
+                      <joint name="arm_7_joint" pos="0 0 0" axis="0 0 1" range="-2.0944 2.0944" damping="1"
```

Comment:
Remove all the pos="0 0 0" in joint definitions.

Reply:
Done